src/GitHub.Exports/Models/GitReferenceModel.cs

```diff
@@ -5,25 +5,26 @@ namespace GitHub.Models
 {
     public class GitReferenceModel
     {
-        public GitReferenceModel(string @ref, string label, string repositoryCloneUri)
-            : this(@ref, label, new UriString(repositoryCloneUri))
+        public GitReferenceModel(string @ref, string label, string sha, string repositoryCloneUri)
+            : this(@ref, label, sha, new UriString(repositoryCloneUri))
         {
         }
 
-        public GitReferenceModel(string @ref, string label, UriString repositoryCloneUri)
+        public GitReferenceModel(string @ref, string label, string sha, UriString repositoryCloneUri)
         {
             Guard.ArgumentNotEmptyString(@ref, nameof(@ref));
             Guard.ArgumentNotEmptyString(label, nameof(label));
-            Guard.ArgumentNotNull(repositoryCloneUri, nameof(repositoryCloneUri));
-            Guard.ArgumentNotEmptyString(repositoryCloneUri.ToString(), nameof(repositoryCloneUri));
+            Guard.ArgumentNotEmptyString(sha, nameof(sha));
```

Comment:
https://github.com/github/VisualStudio/blob/grokys/maintainer-workflow-diff/src/UnitTests/Helpers/TestBaseClass.cs#L72 needs to be updated to not default to an empty string, it breaks any tests that create PRs like `ExpiredIndexClearsItems`.

Tests are hanging on VS on my branch because exceptions are getting thrown and our test observables aren't catching them, so they hang (and xunit doesn't catch them, not sure what's up with that). Do tests pass locally for you?